addons/digest/data/digest_template_data.xml

```diff
@@ -3,8 +3,8 @@
     <record id="digest_mail_template" model="mail.template">
         <field name="name">Digest: Default main template</field>
         <field name="model_id" ref="digest.model_digest_digest"/>
-        <field name="email_from">${user.email_formatted | safe}</field>
         <field name="subject">${'%s: %s' % (ctx.get('user', user).company_id.name, object.name)}</field>
+        <field name="email_from">${('"%s" &lt;%s&gt;' % (object.company_id.name, object.company_id.email or user.email_formatted))|safe}</field>
```

Comment:
I think you could use object.company_id.partner_id.email_formatted or user.email_formatted. Could you give a try ?

Reply:
Yep. Works! Done :+1: